docs/vectorizer-worker.md

```diff
@@ -82,48 +82,25 @@ On your local machine:
    - Docker: `docker compose exec -it db psql`
    - psql:  `psql postgres://postgres:postgres@localhost:5432/postgres`
 
-### Install the vectorizer worker Docker image
+### Standalone vectorizer worker with Docker
 
-This docker image supplies a pgai image with vectorizer worker. You use this image to 
-run vectorizers on any self-hosted Postgres database that has the pgai extension activated.
+The `timescale/pgai-vectorizer-worker` docker image supplies the pgai vectorizer worker.
+You use this image to run vectorizers on any self-hosted Postgres database that has the
+pgai extension activated.
 
 On your local machine:
 
-1. **Create the Docker configuration for pgai vectorizer worker**
-
-   Copy the following configuration into a file named `docker-compose.yml`:
-   ```yaml
-   name: pgai
-   services:
-     vectorizer-worker:
-       image: timescale/pgai-vectorizer-worker:v0.2.1
-       environment:
-         PGAI_VECTORIZER_WORKER_DB_URL: postgres://<username>:<password>@<host>:<port>/<database-name>
-         OPENAI_API_KEY: <your-api-key>
-         OLLAMA_HOST: <ollama-host-url>
-         PGAI_VECTORIZER_OLLAMA_MAX_CHUNKS_PER_BATCH: <ollama-max-chunks-per-batch>
-   ```
-
-  1. Replace the values of:
-     - `PGAI_VECTORIZER_WORKER_DB_URL`: the postgres connection string to the database where you have defined vectorizers.
-     - `OPENAI_API_KEY`: with a key from your AI provider (if using OpenAI).
-     - `OLLAMA_HOST`: with the base url of your ollama installation (if using Ollama and the url is not http://localhost:11434).
-     - `PGAI_VECTORIZER_OLLAMA_MAX_CHUNKS_PER_BATCH`: with the max chunks per batch (if not 2048).
-
 1. **Run the vectorizer worker**
 
    For self-hosted, you run a pgai vectorizer worker to automatically create embedding from the data in your
    database using [vectorizers you defined previously](/docs/vectorizer.md#define-a-vectorizer).
 
-   In a new terminal, start the vectorizer worker:
-   ```shell
-   docker compose up -d
+   Start the vectorizer worker:
```

Comment:
Double-checking here. I seem to remember that the new terminal was necessary.

Reply:
No longer